### ISSUE TITLE
Refresh controls

### ISSUE THREAD

**Issue by scottybo** [danielemoraschi/maplace.js#94]
I have a number of filter buttons which dynamically update the map locations. I'm using the code below to remove all the locations and then load the map with new locations. The problem is that the "list" control which shows the place names doesn't reload, and a new list box appears beneath the existing box.

Is there a way to "reload" the controls div?

``` javascript
if (Maplace.Loaded()) {
    Maplace.RemoveLocations(1);
    Maplace.Load({locations: NewLocations});
} else {
    Maplace.Load({locations: Locations});
}
```

**Comment by danielemoraschi** [comment]
Hi,
you should save the reference of your object into a variable and call the Load function against that, which will be the instance of you map in the page. In your example every time you call Maplace.Load you are actually creating a new instance of maplace.

Try sometime like:

```javascript
var myMap = Maplace.Load({locations: Locations});

if (myMap.Loaded()) {
    myMap.RemoveLocations(1);
    myMap.Load({locations: NewLocations});
} else {
    myMap.Load({locations: Locations});
}
```

**Comment by scottybo** [comment]
Sorry, I don't think I explained myself properly. I had the myMap variable set, but didn't include it in my snippet.

The problem I'm facing is that the "list" control box (the white box which shows a list of all the places) doesn't get updated with the new locations. Instead, it stays on the map and a second list control box appears underneath. Any ideas?

Thanks for such a fast reply by the way :)

**Comment by scottybo** [comment]
Attached a screenshot to help explain better. Blurred out the contents due to sensitive data. The list boxes contain the locations correctly filtered, but it would be good if I could simply replace the content in the existing box. Any ideas?
![maps](https://cloud.githubusercontent.com/assets/11498904/6704786/5efebe0e-cd45-11e4-8a9c-f75252c182d3.jpg)

**Comment by danielemoraschi** [comment]
Can you paste the code you are using in your page please? here or on jsfiddle, as the screenshot is not very useful to me to understand what's happening. Thank you

**Comment by scottybo** [comment]
Sure. Note that "rmv" is the acronym of the project, it's not short for "remove" :)

```javascript
var rmv_locations = [];

jQuery(".mix:visible").each(function(index) { // loop through all the visible mix items
		var rmv_place = {
			lat:jQuery(this).data("lat"),
			lon:jQuery(this).data("lon"),
			title:jQuery(this).data("name"),
			html:jQuery(this).data("name"),
			icon:"https://maps.google.com/mapfiles/markerA.png",
			animation:"google.maps.Animation.DROP"
		}
		rmv_locations.push(rmv_place);
});


if (rmvMap.Loaded()) {
	console.log("map has been previously loaded");
	rmvMap.RemoveLocations(1);
	rmvMap.Load({locations: rmv_locations});
} else {
	console.log("map not previously loaded");
	rmvMap.Load({locations: rmv_locations});
}
```

**Comment by scottybo** [comment]
Still having issues with this one - just wondering if you have any ideas? :)

**Comment by aeroradish** [comment]
@scottybo.  I have a workaround for you.  Right before if (rmvMap.Loaded()), use css hide the control using the following.  

    var theDdl = $('wrap_controls');
    if(thDdl.length >0) {
        theDdl.hide();
    }

**Comment by danielemoraschi** [comment]
Related to #161 and fixed on v0.2.8
